ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -154,6 +154,17 @@ public class KsqlConfig extends AbstractConfig {
   public static final String KSQL_WRAP_SINGLE_VALUES =
       "ksql.persistence.wrap.single.values";
 
+  public static final String KSQL_QUERYANONYMIZER_ENABLED =
+      "ksql.queryanonymizer.logs_enabled";
+  private static final String KSQL_QUERYANONYMIZER_ENABLED_DOC =
+      "This defines whether we log anonymized queries out of query logger or if we log them"
+      + "in plain text. Defaults to true";
+  public static final String KSQL_QUERYANONYMIZER_CLUSTER_NAMESPACE =
+      "ksql.queryanonymizer.cluster_namespace";
+  private static final String KSQL_QUERYANONYMIZER_CLUSTER_NAMESPACE_DOC =
+      "Namespace used in the query anonymization process, representing cluster id and "
+          + "organization id respectively. For example, 'clusterid.orgid'.";
+
```

Comment:
Additional configs that are not available in 6.2 but where introduced in later versions.